lib/parsers/livesplit.rb

```diff
@@ -85,6 +85,7 @@ def v1_6(xml, fast)
         raise "Not a proper LiveSplit run file"
       end
 
+      run[:history].unshift(run[:time]) unless fast || run[:time].in?(run[:history])
```

Comment:
Not sure if LiveSplit has always tracked segment history. @CryZe / @wooferzfg?

If it has, then yeah that sounds like a reasonable solution if we can navigate around the edge cases like missed splits and resets.